src/EventSubscriber/ExceptionSubscriber.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Acquia\Cli\EventSubscriber;
+
+use Acquia\Cli\Exception\AcquiaCliException;
+use League\OAuth2\Client\Provider\Exception\IdentityProviderException;
+use Symfony\Component\Console\ConsoleEvents;
+use Symfony\Component\Console\Event\ConsoleErrorEvent;
+use Symfony\Component\Console\Event\ConsoleEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class ExceptionSubscriber implements EventSubscriberInterface
+{
+
+  /**
+   * @return array
+   */
+  public static function getSubscribedEvents() {
```

Comment:
Based on https://symfony.com/doc/current/event_dispatcher.html#creating-an-event-subscriber

Reply:
This method gets called.